src/Generator/Commands/ActionGenerator.php

```diff
@@ -76,20 +76,99 @@ protected function getFilePath(): string
 
     protected function getStubFileName(): string
     {
-        return "actions/{$this->stub}.stub";
+        $file = new \Nette\PhpGenerator\PhpFile();
+        $namespace = $file->addNamespace('App\Containers\\' . $this->sectionName . '\\' . $this->containerName . '\Actions');
+        $isCRUD = in_array($this->stub, ['list', 'find', 'create', 'update', 'delete']);
+
+        // imports
+        $parentActionFullPath = 'App\Ship\Parents\Actions\Action';
+        $namespace->addUse($parentActionFullPath, 'ParentAction');
+        if ($isCRUD) {
+            if (in_array($this->stub, ['create', 'update', 'find'])) {
+                $modelFullPath = 'App\Containers\\' . $this->sectionName . '\\' . $this->containerName . '\Models\\' . $this->model;
+                $namespace->addUse($modelFullPath);
+            } elseif ('list' === $this->stub) {
+                $lengthAwarePaginatorFullPath = '\Illuminate\Contracts\Pagination\LengthAwarePaginator';
+                $namespace->addUse($lengthAwarePaginatorFullPath);
+            }
+            $taskFullPath = 'App\Containers\\' . $this->sectionName . '\\' . $this->containerName . '\Tasks\\' . Str::ucfirst($this->stub) . $this->model . ('list' === $this->stub ? 's' : '') . 'Task';
+            $namespace->addUse($taskFullPath);
+            $requestFullPath = 'App\Containers\\' . $this->sectionName . '\\' . $this->containerName . '\UI\\' . $this->ui . '\Requests\\' . Str::ucfirst($this->stub) . $this->model . ('list' === $this->stub ? 's' : '') . 'Request';
+            $namespace->addUse($requestFullPath);
+        }
+
+        // class
```

Comment:
all commented parts like this should just be their own methods.

Reply:
Currently working on this ...